zipkin-server/src/main/java/zipkin2/server/internal/ZipkinQueryApiV2.java

```diff
@@ -156,6 +157,7 @@ public AggregatedHttpMessage getAutocompleteValues(@Param("key") String key) thr
    * active use, and active use usually implies more than 3 services.
    */
   AggregatedHttpMessage maybeCacheNames(boolean shouldCacheControl, List<String> values) {
+    Collections.sort(values);
```

Comment:
did a quick check and I don't think any backends return immutable lists. so this should be fine